pastas/__init__.py

```diff
@@ -36,7 +36,7 @@
     TarsoModel,
     WellModel,
 )
-from .timeseries import TimeSeries
+from .timeseries import TimeSeries, validate_stress, validate_oseries
```

Comment:
Do we want to expose TimeSeries directly?

Reply:
Good point, this should not be necessary anymore. I will remove this import.